k8s/kubeval.bzl

```diff
@@ -0,0 +1,73 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An implementation of kubeval for validating k8s objects."""
+
+def _impl(ctx):
+  """Core implementation of kubeval."""
+  content="""
+#!/bin/bash
+set -e
+
+# The schema directory has to have the structure 'kubernetes-json-schema/master', so do that in a tmpdir
+tmp_dir=$(mktemp -d)
+trap "{{ rm -rf $tmp_dir; }}" EXIT
+
+schema_dir=$tmp_dir/kubernetes-json-schema/master/
+mkdir -p $schema_dir
+ln -s $(pwd)/external/kubeval_schemas/* $schema_dir
+
+cat {config} | {kubeval} --schema-location=file://$tmp_dir --kubernetes-version={kube_version}
```

Comment:
We use `.sh.tpl` files for all the other actions.

Reply:
Done.